roles/easy-build-packages/defaults/main.yml

```diff
@@ -1 +1,2 @@
-sm_files_path: "{{ sm_prefix }}/easybuild_files"
\ No newline at end of file
+sm_files_path: "{{ sm_prefix }}/easybuild_files"
+sm_files_repo_version: "4ef7ae6cc2284f69412a8db5e10dddd92024eeab"
```

Comment:
What's up with this repo_version?

Reply:
Most recent commit in https://github.com/DeepOps/easyconfigs . `ansible-lint` doesn't like `HEAD` as a git checkout target.

Reply:
Okay, sounds good. I also prefer pointing to a specific hash.